app/facades/foods_facade.rb

```diff
@@ -0,0 +1,18 @@
+class FoodsFacade
```

Comment:
To answer your question about total hits: What if your facade returned data for both the 10 foods AND total hits. This could look a couple different ways -- a parent PORO that stores all the foods and the total hits, or even just a simple hash you make on the spot to return both back to the controller.